lib/code_corps/messages/messages.ex

```diff
@@ -53,6 +53,10 @@ defmodule CodeCorps.Messages do
     Conversation |> Repo.get(id)
   end
 
+  def update_conversation(conversation, params) do
+    Conversation.update_changeset(conversation, params) |> Repo.update
+  end
```

Comment:
Style thing but I would consider leading with the pipe, so `conversation |> Conversation.update_changeset`...